src/XTMF2/Bus/ClientBus.cs

```diff
@@ -31,42 +32,48 @@ namespace XTMF2.Bus
     /// </summary>
     public sealed class ClientBus : IDisposable
     {
-        private readonly Stream _ClientHost;
-        private readonly bool _Owner;
-        private volatile bool _Exit = false;
+        private readonly Stream _clientHost;
+        private readonly bool _owner;
+        private volatile bool _exit = false;
 
-        private readonly Scheduler _RunScheduler;
+        private readonly Scheduler _runScheduler;
+        private readonly List<string> _extraDlls;
 
         /// <summary>
         /// The link to the XTMFRuntime
         /// </summary>
         public XTMFRuntime Runtime { get; private set; }
 
+        /// <summary>
+        /// Additional DLLs that the client should load.
+        /// </summary>
+        public IReadOnlyList<string> ExtraDlls => _extraDlls;
+
         /// <summary>
         /// Create the bus to interact with the host.
         /// </summary>
         /// <param name="serverStream">A stream that connects to the host.</param>
         /// <param name="streamOwner">Should this bus assume ownership over the stream?</param>
         /// <param name="runtime">The XTMFRuntime to work within.</param>
-        public ClientBus(Stream serverStream, bool streamOwner, XTMFRuntime runtime)
+        public ClientBus(Stream serverStream, bool streamOwner, XTMFRuntime runtime, List<string> extraDlls = null)
         {
             Runtime = runtime;
-            _RunScheduler = new Scheduler(this);
-            _ClientHost = serverStream;
-            _Owner = streamOwner;
-            Runtime.ClientBus = this;
+            _runScheduler = new Scheduler(this);
+            _clientHost = serverStream;
+            _owner = streamOwner;
+            _extraDlls = extraDlls;
```

Comment:
Wondering if it makes more sense to store an empty list rather than null if there are no extraDlls

Reply:
That makes sense.  Creating a patch for it now.